### DISCUSSION TITLE
API to know test results?

### DISCUSSION THREAD

**Discussion by hhimanshu** [codesandbox/codesandbox-client#5595] [Q&A] [answered]
Hello team,

I see that Codesandbox supports an API https://codesandbox.io/docs/api, however, I am not sure if today, I can get the results for tests runs in a sandbox. Is this is possible today?

How can I achieve this goal with CodeSandbox?

Thanks

**Comment by andrasbacsai** [comment, marked as answer]
Hey,

You would like to get the frontend test results? If so, it's not possible through our API.

**Reply by hhimanshu** [reply]
I see, would you guide me where the change should be made if that API were to be available? I would be happy to make the change. Thanks

**Reply by andrasbacsai** [reply]
I'm sorry, but our API is not open-sourced.

**Reply by harit-h** [reply]
But the sandbox can run completely in the browser using `client` environment. I thought some module should be able to give access to the test output. Isn't all the tests running on the client? (or are they running on the server?)
